src/components/Card/Card.test.tsx

```diff
@@ -1,7 +1,7 @@
 import { screen, render } from '@testing-library/react';
 import { Card } from './Card';
 import { ICard } from '../../types/interfaces';
-import database from '../../database/source.json';
+import database from '../../Api/mockSource.json';
```

Comment:
better create separate folder 'mocks' where u will collect all your mock objects and functions